spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -6,16 +6,23 @@
     { target: FactoryBot.attributes_for(:target, user_id: user.id, topic_id: topic.id) }
   end
 
-  let(:user)       { create(:user_with_devices) }
-  let(:user_match) { create(:user_with_devices) }
-  let(:topic)      { create(:topic) }
+  let(:user)               { create(:user_with_devices) }
+  let(:user_match)         { create(:user_with_devices) }
+  let(:topic)              { create(:topic) }
+  let(:topic_other_target) { create(:topic) }
 
   let(:target) do
     create(:target, user: user_match, latitude: params[:target][:latitude] + 0.0001,
                     longitude: params[:target][:longitude] + 0.0001,
                     topic_id: params[:target][:topic_id])
   end
 
+  let(:target_same_title) do
```

Comment:
I would move this `let` inside the context where it is used (`context 'same title but from other user'`)